doc/whatsnew.rst

```diff
@@ -13,6 +13,7 @@ Changes to :doc:`/api/tools-costs`
   - Change the default fixed O&M reduction rate to 0 (:pull:`186`).
   - Modify to use 2023 release of IEA WEO data and to use 2022 historic data for the base year (:pull:`187`).
   - Change the default final year to 2110 (:pull:`190`).
+  - Add option to use or not use vintages for fixed O&M costs (:pull:`195`).
```

Comment:
It's possible to link directly to the new Config class attribute:
```rst
- Add :attr:`~.costs.Config.use_vintages` to control whether vintages are used in computing fixed O&M costs (:pull:`195`).
```

This lets the changelog entry be brief, and the documentation of the item *per se* to contain full detail.

Reply:
Updated, thanks for the suggestion!